app/assets/javascripts/controllers/ems_common/ems_common_form_controller.js

```diff
@@ -182,11 +182,12 @@ ManageIQ.angular.app.controller('emsCommonFormController', ['$http', '$scope', '
   $scope.isBasicInfoValid = function() {
     if(($scope.currentTab == "default" && $scope.emsCommonModel.emstype != "azure") &&
       ($scope.emsCommonModel.emstype == "ec2" ||
-      ($scope.emsCommonModel.emstype == "openstack" && $scope.emsCommonModel.default_hostname ||
+       $scope.emsCommonModel.emstype == "openstack" && $scope.emsCommonModel.default_hostname ||
        $scope.emsCommonModel.emstype == "scvmm" && $scope.emsCommonModel.default_hostname ||
        $scope.emsCommonModel.emstype == "openstack_infra" && $scope.emsCommonModel.default_hostname ||
        $scope.emsCommonModel.emstype == "rhevm" && $scope.emsCommonModel.default_hostname ||
-       $scope.emsCommonModel.emstype == "vmwarews" && $scope.emsCommonModel.default_hostname)) &&
+       $scope.emsCommonModel.emstype == "vmwarews" && $scope.emsCommonModel.default_hostname || 
+       $scope.emsCommonModel.emstype == "vmware_cloud" && $scope.emsCommonModel.default_hostname) &&
       ($scope.emsCommonModel.default_userid != '' && $scope.angularForm.default_userid.$valid &&
```

Comment:
Here, I have also removed unnecessary parenthesis pair. I hope it's fine, otherwise, I'll re-add it.

This part of the code could also be rewritten as

```($scope.emsCommonModel.emsType == "openstack" || $scope.emsCommonModel.emsTyps == "scvmm" || ...) && $scope.emsCommonModel.default_hostname ...```